docs/api-docs/orders/orders-api-overview.md

```diff
@@ -611,7 +611,7 @@ Order `subtotal` and `total` calculate automatically; edits to the following pro
 |`billing_address`|
 |`shipping_addresses`|
 
-You can override calculated values such as product prices, subtotals, and totals by sending a fixed value in the request. If values are not supplied for these properties, they will be automatically calculated based on the preset store values and tax rules.
+You can override calculated values such as product prices, subtotals, and totals by sending a fixed value in the request. If you do not supply values for these properties, you will automatically calculate them based on the preset store values and tax rules.
```

Comment:
I think that passive voice is fine in this instance since users do not calculate values themselves.